client/common/sass/components/_text-link.sass

```diff
@@ -18,3 +18,41 @@
 		&:hover
 			background-color: transparent
 			text-decoration: underline
+
+	&--category
+		&::before
+			vertical-align: middle
+			margin-right: 4px
+
+		&.arrest::before
```

Comment:
So one thing I'm realizing here is that the category page `slug` fields in the dev data do not match the ones elsewhere. On staging, they are:

* `arrest-criminal-charge`
* `border-stop`
* `chilling-statement`
* `denial-access`
* `equipment-damage`
* `equipment-search-seizure-or-damage`
* `leak-case`
* `other-incident`
* `physical-attack`
* `prior-restraint`
* `subpoena`

This makes me wonder if it might be wise to de-couple of the name of the CSS class from the page slug? Like, maybe it could be a separate field that defines a set list of choices. Maybe though that's overdoing it if the slugs aren't expected to change at all, and we can update the dev data to match what's in prod/staging. On the other hand, I guess it depends a little on how we're going to use these icons, and if this is only confusing because of how we're implementing showing a complete icon list on the Styleguide page, and maybe elsewhere the icon class names will not be loaded from the CategoryPage model object at all.

What do you think about this? I think you are more familiar with the use of the icons across the whole site as described in the Figma design.

Reply:
Yes. I was giving this thought as well. I am guessing, everytime we will have a CategoryPage model object when we are trying to show a category link itself? For the rest of the places like filters, etc., I feel the implementation for the rest places apart from the category link will need to be separate from this anyways.

As of the field, I was also thinking if it makes sense to add a separate field, but then again since there was already a unique field, I just wanted to use slug. I feel we can generalize to use the same slug everywhere? I am not sure how often we plan to change slugs, but if not very often, I would just keep using that. But if we plan to change slug pretty often, I think we can add a separate field. @harrislapiroff any ideas? I definitely think adding a separate field is probably the safe way to go about it.